test/performance/odp_l2fwd.c

```diff
@@ -1280,6 +1277,9 @@ static int create_pktio(const char *dev, int idx, int num_rx, int num_tx, odp_po
 	       "%02x:%02x:%02x:%02x:%02x:%02x\n", dev, info.drv_name, num_rx, num_tx,
 	       addr[0], addr[1], addr[2], addr[3], addr[4], addr[5]);
 
+	if (gbl_args->appl.verbose)
+		odp_pktio_print(pktio);
+
```

Comment:
Could still be printed before the above /dev/drv/queue/mac summary to preserve the previous ordering?

Reply:
I prefer this order because now `printf("  dev: %s, drv: %s...` print works as heading  for `odp_pktio_print()` output, which may be quite long and the interface name may not be obvious.